pkg/codegen/internal/test/Makefile

```diff
@@ -0,0 +1,33 @@
+.PHONY = update_random update_aws update_azure update_azure_native update_kubernetes
```

Comment:
I'd consider rewriting this makefile in Go can be very much worth it, it is a little extra effort but is much more easily cross-platform.

Reply:
Is this actually used cross-platform? I'd prefer not to change the way we build things for one specific case.

Reply:
(even if it is used cross-platform, I don't think that this is the right time or place to add more ways that SW gets built in this repo--I'd much rather stick with the status quo for now)